.travis.yml

```diff
@@ -19,5 +19,5 @@ jobs:
       os: linux
       script:
         - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate();'
-        - julia --project=docs/ docs/make.jl
+        - travis_wait 20 julia --project=docs/ docs/make.jl
```

Comment:
this change is unrelated to your PR; please rebase with master

Reply:
done